lib/Artwork.php

```diff
@@ -665,6 +688,7 @@ public function Save(): void{
 			CompletedYear = ?,
 			CompletedYearIsCirca = ?,
 			Created = ?,
+			Updated = ?,
```

Comment:
We don't have to set the Updated column because it's of type `timestamp`, which the DB automatically updates any time the row changes.

Reply:
Thanks for explaining that. I also see it now in the schema. Removed.